services/gcn_service/gcn_service.py

```diff
@@ -53,9 +55,22 @@ def service():
     if notice_types is None or notice_types == '' or notice_types == []:
         log('No notice_types configured to poll gcn events (config: gcn.notice_types')
         return
+
+    group_id = cfg['gcn.group_id']
```

Comment:
would `get('gcn.group_id')` be better here?

Reply:
@guynir42 Good point. It is called group.id in kafka, but I am renaming it as client_group_id.